vscode-trace-extension/package.json

```diff
@@ -258,12 +258,26 @@
                 "trace-compass.traceserver.url": {
                     "type": "string",
                     "default": "http://localhost:8080",
-                    "description": "Enter the trace server's URL, including port. Eg: http://localhost:8080."
+                    "description": "Enter the trace server's URL, including port. Eg: http://localhost:8080.",
+                    "order": 0
                 },
                 "trace-compass.traceserver.apiPath": {
                     "type": "string",
                     "default": "tsp/api",
-                    "description": "Enter the trace server's API path, to be appended to the server URL. Eg: 'tsp/api'."
+                    "description": "Enter the trace server's API path, to be appended to the server URL. Eg: 'tsp/api'.",
+                    "order": 1
+                },
+                "trace-compass.traceserver.enableSeparateBackendUrl": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Enable separate backend URL - Allows the vscode-backend to use a different URL. This is useful for a remote-deployment with K8s where the frontend needs to access an ingress, while backend needs to access a localhost for trace-server on the same machine.",
```

Comment:
I would suggest making it explicit that in the "separate BE URL" mode, that URL is meant to point to the same trace server, not a second one. Maybe a slight rewording would do the trick: I was not sure about the part "on the same machine" - it could be on a different machine I think? If so, what about something like: "[...] while backend needs to access the same trace-server using an internal URL"? Or maybe there would be a better way to express similar?

Reply:
Looks good, thanks!